.cicd/scripts/srw_unittest.sh

```diff
@@ -0,0 +1,33 @@
+#!/user/bin/env bash
+#
+# A test script for running the SRW application unittest tests that
+# should be tested on-prem.
+#
+set -e -u -x
+
+script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" > /dev/null 2>&1
+&& pwd)"
```

Comment:
With the removal of the parallel section, the `Jenkinsfile` is being properly read in.  The new `.cicd/scripts/srw_unittest.sh` script is failing with the following message:

`Syntax error near unexpected token &&.`

Please move the `&& pwd)"` entry on line 9 to line 8.
```suggestion
script_dir="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
```